src/App.tsx

```diff
@@ -95,25 +101,35 @@ function RenderDrawerItem(props, i: number) {
 
 function RenderDrawer(props) {
   var items = [];
-  for (var i = 0; i < RNGalleryList.length; i++) {
-    if (RNGalleryList[i].key !== 'Settings') {
-      items.push(RenderDrawerItem(props, i));
-    }
+  for (var i = 2; i < RNGalleryList.length; i++) {
```

Comment:
The first element in RNGalleryList is the element that opens as the default page upon app opening so that has to stay first. I can change the code here to check in the key is 'Settings' or 'Home' , if that's more clear. Or maybe in order to keep the code more concise I could just add a comment explaining why the iteration is starting at 2.

Reply:
Comment added!